src/collections/resources/articles/terraform-with-meshery/index.mdx

```diff
@@ -30,23 +30,50 @@ import Button from "../../../../reusecore/Button";
 </div>
 
 <p>
-    One way to further enhance your use of Terraform is by integrating it with Meshery. Meshery is a cloud-native management platform that provides a unified interface for managing and monitoring your infrastructure resources, including those managed by Terraform. By integrating Terraform with Meshery, you can leverage the power and flexibility of both tools to streamline your infrastructure management process.
+  One way to further enhance your use of Terraform is by integrating it with
```

Comment:
Just a question, why are we modifying this file by puttig line breaks?

Reply:
Hi @Shivam-AfA,

Thanks for your message and working on this PR.

This file was mainly edited to update the `<ResourceWrapper>`  tag to `<ResourcesWrapper>`
![image](https://github.com/layer5io/layer5/assets/90356410/87c45d05-2dd3-4c6d-9c44-c64283bb44bd)

to match the import statement: 
`import { ResourcesWrapper } from "../../Resources.style.js";` 

and clear a MDX warning that was appearing during builds:
`warn Component ResourceWrapper was not imported, exported, or provided by MDXProvider as global scope`

Not sure if I also made edits to the spacing as part of general page cleanup/tidying.

Feel free to let me know if you would like me to remove the line break edits.

Thanks,
Randy